modules/upgrade/pages/server-y-zypper.adoc

```diff
@@ -23,11 +23,18 @@ zypper migration
 Zypper will show the possible migration targets with detailed summaries.
 . Select the appropriate target, and follow the prompts to complete the migration.
 . Reboot the server.
-. When rebooted the spacewalk services are not running until you have migrated the PostgreSQL database to version{nbsp}12.
-. Log in on the text console as root, and run the database migration script:
+. When rebooted the {productname} spacewalk services are not running until you have migrated the PostgreSQL database to version{nbsp}13.
+. Log in on the text console as root.
+If you are upgrading from 4.1 to 4.2, run the database migration script:
++
+----
+/usr/lib/susemanager/bin/pg-migrate-12-to-13.sh
+----
+. If you are upgrading from 4.0 to 4.2, run the database migration scripts:
 +
 ----
 /usr/lib/susemanager/bin/pg-migrate-10-to-12.sh
+/usr/lib/susemanager/bin/pg-migrate-12-to-13.sh
```

Comment:
same as above. Needs to be checked how this works. Maybe it is easier to provide a 10-to-13 script